offices/_single.html

```diff
@@ -139,55 +146,53 @@ <h1 class="h2">
                     </span>
                 </button>
                 <div class="expandable_content">
-                    {% if initiative.title %}
+                    {% if page.short_title or page.title %}
                     <h2 class="h4 u-hide-on-mobile">
-                        {{ initiative.title | safe }}
+                        {% if page.short_title %}
```

Comment:
Why not just use an or statement like `{{ (page.short_title or page.title) | safe }}`